src/python/gem5/resources/resource.py

```diff
@@ -741,6 +800,7 @@ def obtain_resource(
     # Once we know what AbstractResource subclass we are using, we create it.
     # The fields in the JSON object are assumed to map like-for-like to the
     # subclass contructor, so we can pass the resource_json map directly.
+
```

Comment:
Extra newline?

Reply:
I removed this extra line in the second commit.